test/real_cases_test.rb

```diff
@@ -228,7 +228,7 @@ def test_ortools_global_with_identical_route_duration_and_vehicle_window
       assert_equal 29, (result[:routes].count{ |r| r[:activities].count{ |a| a[:service_id] }.positive? })
 
       # Check total travel time
-      assert result[:routes].map{ |r| r[:total_travel_time] }.reduce(&:+) < 176000, "Too long travel time:#{result[:routes].map{ |r| r[:total_travel_time] }.reduce(&:+)}"
+      assert result[:routes].map{ |r| r[:total_travel_time] }.reduce(&:+) < 178000, "Too long travel time:#{result[:routes].map{ |r| r[:total_travel_time] }.reduce(&:+)}"
```

Comment:
Did you have an issue with this test @fonsecadeline ? 

We haven't touch optim-ortools recently, it is weird :thinking: @braktar do you have any problems with it?

Reply:
It appeared when I removed vehicles useless sort but I could not figure out why...

Reply:
OR-Tools v7.5+ find a solution one less unassigned. which increase the total_travel_time.
So that's ok to increase this value. But additonnaly, the total number of unassigned must be decreased accordingly. Maybe with a note to indicate that the new conditions only pass with OR-Tools v7.5+

Reply:
I don't think the reason here is not related to OR-Tools version, but really to vehicles order. 

Indeed, at previous commit we get same number of unassigned stops but with a lower travel_time.

Every vehicle is identical except from skills. I think first solution strategy is vehicles order dependent and can slightly change solution.

Reply:
Ok :) (that would rather be <= 30 for 7.4 and 7.1, <= 29 for 7.5, I will use 30).

Reply:
Well, in fact there already is a check for unassigned number. How can it be 1 unassigned more with 7.4 ? :p

Reply:
oups, sorry, I did everything at the opposite, 29 is number of assigned stops.

Reply:
Still, checking number of stops in route is equivalent to checking number of unassigned.

Reply:
That's the number of routes I think.

Reply:
If there are more than one service in each vehicle, then checking the number of vehicles is not enough.

Reply:
Yep, I looked too fast :)